data/Alderney/States/sources/manual/terms.csv

```diff
@@ -1,2 +1,3 @@
 id,name,start_date,end_date,source
 2014,2014–2016,2014-11-22,2016-11,
+2017,2017-2018,2017-01-01,2018-12-31,http://www.alderney.gov.gg/article/4386/Steve-Roberts
```

Comment:
I don't think this is a particularly good `source` entry here, as it doesn't obviously say anything more broad than the single member. But generally we're moving away from having `source` entries in this file anyway, so I'd just drop the column. The citation in the commit message is good enough here if someone needs to trace the provenance of this.